src/main/java/seedu/mark/logic/commands/EditAnnotationCommand.java

```diff
@@ -44,7 +44,11 @@
     public static final String MESSAGE_TARGET_NO_PHANTOM = "You cannot move an annotation to a phantom paragraph.";
     public static final String MESSAGE_PHANTOM_CANNOT_HIGHLIGHT = "You cannot change the highlight "
             + "of a phantom paragraph.";
+    public static final String MESSAGE_CANNOT_MOVE_TO_SAME_PARA = "Please specify a different paragraph to move "
+            + "annotation to.";
     public static final String MESSAGE_NOTHING_TO_EDIT = "Paragraph %1$s has no annotations to edit.";
+    public static final String MESSAGE_NO_EDITS_GIVEN = "It seems you forgot to indicate the note or highlight to "
+            + "change! Otherwise there is nothing for me to edit. D:";
```

Comment:
This is cute... but maybe we can match the tone of the other messages?

Reply:
Ok changed!